packages/webcert/src/store/certificate/certificateMiddleware.ts

```diff
@@ -157,6 +157,7 @@ const handleGetCertificateSuccess: Middleware<Dispatch> = ({ dispatch }) => () =
     dispatch(validateCertificate(action.payload.certificate))
   }
   dispatch(getCertificateEvents(action.payload.certificate.metadata.id))
+  dispatch(updateCertificateSignStatus(CertificateSignStatus.INITIAL))
```

Comment:
This resets `CertificateSignStatus` and makes the modal disappear once new certificate data is loaded.